src/main/java/com/celements/course/service/CourseService.java

```diff
@@ -526,10 +524,13 @@ private boolean isParticipantInState(XWikiDocument regDoc, String emailAdr,
         XWikiDocument regDoc = createRegistrationDoc(courseDocRef);
         List<BaseObject> copied = participantObjsToCopy.stream().map(obj -> Pair.of(obj,
             XWikiObjectEditor.on(regDoc).filter(participantClassDef).filter(
-                obj.getNumber()).createFirstIfNotExists())).filter(
-                    pair -> copyDocService.copyObject(pair.getLeft(), pair.getRight())).map(
-                        Pair::getRight).map(restoreParticipantObj(courseDocRef)).collect(
-                            toImmutableList());
+                obj.getNumber()).createFirstIfNotExists()))
+            .filter(
+                pair -> copyDocService.copyObject(pair.getLeft(), pair.getRight()))
+            .map(
+                Pair::getRight)
+            .map(restoreParticipantObj(courseDocRef)).collect(
+                toImmutableList());
```

Comment:
multiple scrambled formattings in this file. are you using our formatter definitions?
https://github.com/celements/synventis-toolbox/tree/dev/eclipse

Reply:
This did not happen in the 5.x branch (CELDEV-964) thus I think it is very unlikly to be the formatter definitions. I cannot tell how this happend.

Reply:
that's weird, yet you've broken the formatting again with the latest commit after fixing it.

Reply:
Fixed the style again. I reimported the formatting. Then
1. chose select all in the editor
2. chose "Format" from Edit-Menu
-> the codemix editor is breaking it again.

Reply:
I cannot reproduce this in my environment, for me no CodeMix formatter runs on java files, only the eclipse integrated java formatter via save action.